frontend/src/console/components/ThemeFooter/ThemeFooter.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import './styles.scss';
+import {
+  ConsolePage,
+  ConsolePageCustomizationContainer,
+  FooterPreview
+} from 'console/components';
+import { InstancesModel } from 'console/models';
+import { connect } from 'react-redux';
+import { RootState } from 'global/state';
+import { updateThemeFieldValue } from 'console/actions';
+import { Col, Row } from 'react-bootstrap';
+import { ColorInputField } from '../../../ui/components/ColorInputField';
+import messages from './displayMessages';
+import { WrappedMessage } from '../../../utils/intl';
+
+interface State {}
+
+interface ActionProps {
+  updateThemeFieldValue: Function;
+}
+
+interface StateProps extends InstancesModel {}
+
+interface Props extends StateProps, ActionProps {}
+
+export class ThemeFooterComponent extends React.PureComponent<Props, State> {
+  private onChangeColor = (fieldName: string, newColor: string) => {
+    const instance = this.props.activeInstance;
+
+    if (instance.data) {
+      this.props.updateThemeFieldValue(instance.data.id, fieldName, newColor);
+    }
+  };
+
+  public render() {
+    const instance = this.props.activeInstance;
+    let themeData;
+
+    if (instance.data && instance.data.draftThemeConfig) {
+      themeData = instance.data.draftThemeConfig;
+    }
+
+    return (
+      <ConsolePage contentLoading={this.props.loading}>
+        <ConsolePageCustomizationContainer>
```

Comment:
There's some alignment issue with this page's content's, can you fix it?
Here is the logos page:
![image](https://user-images.githubusercontent.com/27893385/78279072-5f1c9000-74ed-11ea-9f94-c2869e86f2c0.png)

And here is the footer customization page:
![image](https://user-images.githubusercontent.com/27893385/78279125-722f6000-74ed-11ea-9a01-eae2516b8d05.png)

I just noticed that this is also happening on the navigation customization page:
![image](https://user-images.githubusercontent.com/27893385/78279208-9428e280-74ed-11ea-897c-2a6e31a49d72.png)